Source/WebCore/dom/Element.cpp

```diff
@@ -2578,8 +2573,10 @@ void Element::removedFromAncestor(RemovalType removalType, ContainerNode& oldPar
 
     ContainerNode::removedFromAncestor(removalType, oldParentOfRemovedTree);
 
-    if (hasPendingResources())
-        document().accessSVGExtensions().removeElementFromPendingResources(*this);
+    if (UNLIKELY(hasRareData()) && !elementRareData()->effectiveLang().isNull()) {
+        if (effectiveLangFromAttribute(*this).isNull())
+            elementRareData()->setEffectiveLang(nullAtom());
+    }
```

Comment:
What happened here? I don't think we should be adding this code.

Reply:
Oh, I had a conflict and didn't resolve it right it seems. Thanks for catching.